### PR TITLE
footer background colour changed from teal to purple

### PR DESCRIPTION
**Pull request by parisa-0**
## Description
 

Adjust Footer #9
https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/9

Changed background footer from teal to purple

## Change Type
- [ X] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Related Issue

Adjust Footer #9
https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/9

## Screenshots

Before:
image

After:
image

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [X] I have tested my changes locally.
- [X] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: Gemfile.lock
Comment (lilyndk) [inline Gemfile.lock:1]: @parisa-0 This file should not be included :)

Comment (parisa-0) [reply Gemfile.lock:1]: Thanks @lilyndk, the file has been removed

Comment (lilyndk) [inline Gemfile.lock:1]: file should not be deleted

File: Gemfile.lock [hunk 1]
```diff
@@ -29,6 +29,7 @@ GEM
       ruby2_keywords (>= 0.0.4)
     faraday-net_http (3.0.2)
     ffi (1.15.5)
+    ffi (1.15.5-x64-mingw-ucrt)
```
Comment (lilyndk) [inline Gemfile.lock:32]: undo this change

File: Gemfile.lock [hunk 2]
```diff
@@ -255,6 +256,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-21
+  arm64-darwin-23
```
Comment (lilyndk) [inline Gemfile.lock:259]: undo this change
